Modules/_ssl.c

```diff
@@ -75,6 +75,36 @@
 
 
 
+#ifdef BIO_get_ktls_send
+#ifdef MS_WINDOWS
+    typedef long long Py_off_t;
+#else
+    typedef off_t Py_off_t;
+#endif
+
+static int
+Py_off_t_converter(PyObject *arg, void *addr)
+{
+#ifdef HAVE_LARGEFILE_SUPPORT
+    *((Py_off_t *)addr) = PyLong_AsLongLong(arg);
+#else
+    *((Py_off_t *)addr) = PyLong_AsLong(arg);
+#endif
+    if (PyErr_Occurred())
+        return 0;
+    return 1;
```

Comment:
```suggestion
    return PyErr_Occurred() ? 0 : 1;
```

(so that we have an explicit 0/1)